src/MarketingPageAcceptanceTests/MarketingPageAcceptanceTests.Actions/Utils/SqlHelper.cs

```diff
@@ -37,11 +37,48 @@ internal static string GetSolutionAboutLink(string solutionId, string connection
             return result;
         }
 
+        public static int GetSolutionStatus(string solutionId, string connectionString)
+        {
+            var query = "SELECT SupplierStatusId from [dbo].[Solution] where Id=@solutionId";
+            
+            SqlParameter[] parameters = new SqlParameter[] {
+                new SqlParameter("@solutionId", solutionId)
+            };
+
+            var result = SqlReader.Read<int>(connectionString, query, parameters, GetSupplierStatus);
+
+            return result;
```

Comment:
could be 1 thing, more of a personal preference, not necessary.

Reply:
Reason it isn't a thing is to more easily put debugging calls there and detect issues faster